utils/litemap/src/lib.rs

```diff
@@ -34,8 +34,23 @@ extern crate std;
 
 extern crate alloc;
 
+use alloc::vec::Vec;
+use core::marker::PhantomData;
+
 mod map;
 #[cfg(feature = "serde")]
 mod serde;
+pub mod store;
+
+pub type LiteMap<K, V> = map::LiteMapWithStore<K, V, Vec<(K, V)>>;
```

Comment:
thought: should we be using a defaulted type parameter instead?

Reply:
Done.